docs/tutorials/slack-history.md

```diff
@@ -0,0 +1,90 @@
+
+Specifically we will export messages from your slack instance into search engine called MeiliSearch. We will be focusing on getting this setup running from your local workstation. We will mention at the end how you can set up a more productionized version of this pipeline.
+
+We want to make this process easy, so while we will link to some external documentation for further exploration, we will provide all the instructions you need here to get this up and running.
+
+## Set up Airbyte
+
+Make sure you have Docker and Docker Compose installed. Then run the following commands:
+
+```text
+git clone https://github.com/airbytehq/airbyte.git
+cd airbyte
+docker-compose up
+```
+
+If you run into any problems, feel free to check out our more extensive [getting started](../getting-started.md) for more help.
+
+Once you see an Airbyte banner, the UI is ready to go at [http://localhost:8000/](http://localhost:8000/). You can set your user preferences and then you will be brought to a page that asks you to set up a source. In the next step we'll go over how to do that.
+
+
+## Connect to Slack
+
+In the Airbyte UI select Slack from the dropdown. We provide step-by-step instructions for setting up the Slack source in Airbyte [here](../integrations/sources/slack.md). These will walk you through how to complete the form on this page.
+
+By the end of these instructions you should have created a Slack source in the Airbyte UI. The Airbyte app will now prompt you to set up a destination. Next, we will now walk through how to set up MeiliSearch.
+
+## Set up MeiliSearch
+
+First let's get MeiliSearch running on our workstation. MeiliSearch has extensive docs for [getting started](https://docs.meilisearch.com/learn/tutorials/getting_started.html#download-and-launch). For this tutorial, however, we will give you all the instructions you need to set up MeiliSearch using docker.
+
+```bash
+    docker run -it --rm \
+        -p 7700:7700 \
+        -v $(pwd)/data.ms:/data.ms \
+        getmeili/meilisearch
+```
+
+That's it!
+
+{% hint style="info" %}
+MeiliSearch stores data in `$(pwd)/data.ms`, so if you prefer to store it somewhere else, just adjust this path.
+{% endhint %}
+
+## Connect to Airbyte to MeiliSearch
+
+Head back to the Airbyte UI. It should still be prompting you to set up a destination. Select "MeiliSearch" from the dropdown. For the `host` field, if you are on mac set: `http://host.docker.internal:7700`. If you are on linux use `http://localhost:7700`. The `api_key` can be left blank.
+
+## Set up replication
+
+On the next page, you will be asked to select which streams of data you'd like to replicate. We recommend selecting all of them except "files" and "remote files" (you won't really be able to search them well in this search engine).
+
+For frequency, we recommend daily.
```

Comment:
why not hourly?

Reply:
i figure the only reason you really want to use this search is if you are looking for a message that has already been deleted by slack. otherwise it's always going to be more convenient to just search in slack. thus the replication frequency stops mattering too much.